src/bin/cql-stress-scylla-bench/args.rs

```diff
@@ -330,7 +330,7 @@ where
         // Zero means unlimited tries,
         // and #tries == #retries + 1,
         // therefore just subtract with wraparound and treat u64::MAX as infinity
-        let max_retries_per_op = max_errors_at_row.get().wrapping_sub(1);
+        let max_consecutive_errors_per_op = max_errors_at_row.get().wrapping_sub(1);
```

Comment:
I don't understand what the comment about retries is trying to say, previously it made some sense, as the variable used to specifiy the number of retries, but now it would be good to update it so that it explains the new variable.

Reply:
The test `test_retrying()` sets `max_consecutive_errors_per_op = 0`, and then runs an operation which fails the first time it's executed. It's expected that the run will fail.

With `sub(1)` this would correspond to `error-at-row-limit = 1`. That would mean that `error-at-row-limit = 1` doesn't allow any errors, this sounds wrong.